src/components/Dissolution/DissolutionStatement.vue

```diff
@@ -38,9 +60,6 @@ import { DissolutionStatementTypes } from '@/enums'
 
 @Component
 export default class DissolutionStatement extends Vue {
-  @Prop({ default: false })
-  readonly showErrorSummary: boolean
```

Comment:
Was only used by v-card, which I moved to the parent so that this component is pure rows/cols.